crates/air-lib/air-parser/src/parser/air_utils/triplet.rs

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2021 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use crate::ast::FunctionPart;
+use crate::ast::PeerPart;
+use crate::ast::Triplet;
+
+/// Build a `Triplet` from `Call`'s `PeerPart` and `FunctionPart`
+pub(crate) fn try_to_raw_triplet<'i>(
+    peer: PeerPart<'i>,
+    f: FunctionPart<'i>,
+) -> Option<Triplet<'i>> {
+    use FunctionPart::*;
+    use PeerPart::*;
+
+    let (peer_pk, service_id, function_name) = match (peer, f) {
```

Comment:
usually:
pk = public key
sk = secret key

private key is avoided due to pk/pk ambiguity

Reply:
But maybe it makes sense to name it `PeerId` since that's what we actually use.

Reply:
It's kind old outdated name that came from the first release of AquaVM and still with as for the infinity of centuries. I agree that it's highly unclear, but let's rename it in a separe PR, since we have similar names in SDK and somewhere else.